cleo/commands/command.py

```diff
@@ -16,25 +18,28 @@
 
 
 if TYPE_CHECKING:
+    from typing import Literal
+
+    from cleo.io.inputs.argument import Argument
+    from cleo.io.inputs.option import Option
     from cleo.io.io import IO
     from cleo.ui.progress_bar import ProgressBar
     from cleo.ui.progress_indicator import ProgressIndicator
     from cleo.ui.question import Question
+    from cleo.ui.table import Table
+    from cleo.ui.table import _Rows
 
 
 class Command(BaseCommand):
 
-    arguments = []
-    options = []
-
-    aliases = []
-
-    usages = []
-
-    commands = []
+    arguments: list[Argument] = []
+    options: list[Option] = []
+    aliases: list[str] = []
+    usages: list[str] = []
+    commands: list[BaseCommand] = []
 
     def __init__(self) -> None:
-        self._io: IO | None = None
+        self._io: IO = None  # type: ignore[assignment]
```

Comment:
Gonna come back to remove this later, but it was leading to `assert self._io is not None` everywhere

Reply:
Yeah, I was thinking about going as far from throwing `None`s around as possible in a long run.